bundler/lib/bundler/cli/info.rb

```diff
@@ -66,8 +66,19 @@ def print_gem_info(spec)
       gem_info << "\tBug Tracker: #{metadata["bug_tracker_uri"]}\n" if metadata.key?("bug_tracker_uri")
       gem_info << "\tMailing List: #{metadata["mailing_list_uri"]}\n" if metadata.key?("mailing_list_uri")
       gem_info << "\tPath: #{spec.full_gem_path}\n"
-      gem_info << "\tDefault Gem: yes" if spec.respond_to?(:default_gem?) && spec.default_gem?
+      gem_info << "\tDefault Gem: yes\n" if spec.respond_to?(:default_gem?) && spec.default_gem?
+      gem_info << "\tReverse Dependencies: \n\t\t#{gem_dependencies.join("\n\t\t")}" unless gem_dependencies.empty?
```

Comment:
Not sure if this is breaking rubocop, anyway I think it is preferred to use `if gem_dependencies.any?` here. Sorry for this nitpicking.

Reply:
@simi not nitpicking at all!  You're definitely right.  It has been changed 👍